### PR TITLE
W-12559775 config ha incorp feedback duke

### PR DESCRIPTION
**Pull request by dukesphere**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc [hunk 1]
```diff
@@ -253,25 +246,6 @@ To use autoscaling, enable Horizontal Pod Autoscaler (HPA), and configure other
 | `autoscaling.maxReplicas` | The maximum number of replicas that the HPA scaler is allowed to create. Defaults to `11`.
 | `autoscaling.targetCPUUtilizationPercentage` | A resource metric that sets the average CPU usage percentage of all deployed Pods. Defaults to `50`.
 | `autoscaling.targetMemoryUtilizationPercentage` | A string representing the average memory usage percentage of all deployed Pods. Defaults to `nil`.
-| `autoscaling.extraMetrics` | A list of custom Pod configuration metrics. When a Pod runs with these settings, you can collect and monitor application-specific metric data. Defaults to `[]`. Define the metrics with nested parameters, for example:
```
Comment (dukesphere) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc:256]: We're removing this for a few reasons:
* HPA config from our example duplicates the config from our autoscaling.targetCPUUtilizationPercentage setting
* 99% of autoscaling use cases solved by our autoscaling.targetCPUUtilizationPercentage and autoscaling.targetMemoryUtilizationPercentage settings
* providing other extraMetrics settings (without duplicating) is covered in the k8 docs and per Rodrigo G, "user need to setup and add special configuration and maybe some components to the cluster"

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc [hunk 2]
```diff
@@ -131,13 +131,6 @@ autoscaling:
   maxReplicas: 11
   targetCPUUtilizationPercentage: 50
   targetMemoryUtilizationPercentage: 50
-  extraMetrics:
```
Comment (dukesphere) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc:134]: see note on removal below

Comment (dukesphere) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc:134]: @garciarodrigor here's what I removed 👇

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc (hunk 1 above)
Comment (dukesphere) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc:256]: same here 👇 @garciarodrigor

Comment (garciarodrigor) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc:256]: LGTM 👍

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc (hunk 2 above)
Comment (garciarodrigor) [reply gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc:134]: LGTM 👍

**Review by garciarodrigor — APPROVED** [review]
LGTM 👍

**Comment by dukesphere** [comment]
this PR just removes some content, no new content

**Pull request merged**